osf/management/commands/data_storage_usage.py

```diff
@@ -396,11 +397,13 @@ def write_summary_data(filename, summary_data, remote_base_folder):
     file_path = '{}{}'.format(TEMP_FOLDER, filename)
     old_remote = requests.get(
         url=remote_base_folder['files'],
+        headers={'Accept': 'application/vnd.api+json;version={}'.format(DEFAULT_API_VERSION)},
         auth=bearer_token_auth(DS_METRICS_OSF_TOKEN),
         params={'filter[name]': filename},
     ).json()
     try:
-        if old_remote['meta']['total'] > 1:
+        logger.debug('json: {}'.format(old_remote))
+        if old_remote[u'meta'][u'total'] > 1:
```

Comment:
unicode?

Reply:
¯\_(ツ)_/¯ When I updated the version, it said that 'meta' wasn't a key, which is was, so I made the keys unicode (since that's what the response had) and it worked. I'm not sure why, exactly.

Reply:
weird, old_remote['meta']['total'] works for me after your query param version change

Reply:
Same, but not with the `accept` header version (the one that actually works, not the `accept-header` version that doesn't).

Reply:
hm, ok. as long as it's in a working state, I'm fine with it, not going to sort through `accept` and `accept-header` right now.